crates/console-timer/src/lib.rs

```diff
@@ -0,0 +1,75 @@
+/*!
+
+Measuring invocation time on the Web can be done with `console.time`
+and `console.timeEnd`.
```

Comment:
Nitpick: can we link to MDN for more context here? https://developer.mozilla.org/en-US/docs/Web/API/console#Timers

And also mention that the timings end up in the browser's developer tools console and sometimes their performance view? I think we can crib a lot of this text from the rustwasm book: https://rustwasm.github.io/docs/book/reference/time-profiling.html#the-consoletime-and-consoletimeend-functions